employee-rostering-frontend/src/store/types.ts

```diff
@@ -63,6 +64,7 @@ export interface AppState {
   readonly spotList: SpotList;
   readonly contractList: ContractList;
   readonly employeeList: EmployeeList;
+  readonly shiftTemplateList: ShiftTemplateList;
```

Comment:
Needs to be added to all `AppState` objects. There are some in tests. For example in `alerts.test.ts`. Try running `tsc` in `employee-rostering-frontend` if your IDE doesn't report these compilation errors.